app/config/default.json

```diff
@@ -17,12 +17,11 @@
     "port": "8080"
   },
   "serviceClient": {
-    "commonServices": {
-      "ches": {
-        "endpoint": "https://ches-dev.apps.silver.devops.gov.bc.ca/api"
-      },
-      "tokenEndpoint": "https://dev.oidc.gov.bc.ca/auth/realms/jbd6rnxw/protocol/openid-connect/token",
-      "username": "MSSC_SERVICE_CLIENT"
+    "ches": {
+      "endpoint": "https://ches-dev.api.gov.bc.ca/api",
+      "username": "CDACC9DF-012B285B6DB",
+      "tokenEndpoint": "https://dev.loginproxy.gov.bc.ca/auth/realms/comsvcauth/protocol/openid-connect/token"
```

Comment:
Given a choice about how we've been doing things, it's better we don't default this to specific authentication environments, especially if we're still supporting the idea of 3rd parties using CHES for their own needs.